system/security/plugins/jwtSecurity/jwtSecurity.cpp

```diff
@@ -125,6 +125,13 @@ class CJwtSecurityManager : implements IDaliLdapConnection, public CBaseSecurity
 
         if (secretsName.empty())
             throw makeStringException(-1, "CJwtSecurityManager: secretsName not found in configuration");
+
+        // Grab a copy of the name of the internal file scope
+        hpccInternalScope = queryDfsXmlBranchName(DXB_Internal);
+        if (!hpccInternalScope.empty())
```

Comment:
minor: since queryDfsXmlBranchName cannot return empty, I think this conditional makes the code less clear (it implies that that queryDfsXmlBranchName(DXB_Internal) might return empty).

Reply:
Fixed, as well as the bonehead no-op within the if() block.